src/shared/proto/tbots_robot_msg.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto2";
+
+import "shared/proto/robot_component_status.proto";
+
+message TbotsRobotMsg
```

Comment:
Should we add a test that builds this message with all the fields required for the old robots and verifies that it doesn't go over 100 bytes like Gareth suggested? I feel like we are getting pretty close if not already over the limit

Reply:
No, the current plan is to keep the current radio annunciator and convert those messages into TbotsRobotMsg in the RadioBackend

Reply:
Ok that’s fair